src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -329,15 +329,16 @@ public override Task<bool> HandleRequestAsync()
                 RedirectUri = BuildRedirectUri(Options.CallbackPath),
                 Resource = Options.Resource,
                 ResponseType = Options.ResponseType,
-                Prompt = Options.Prompt,
-                Scope = string.Join(" ", Options.Scope)
+                Prompt = properties.GetParameter<string>(OpenIdConnectParameterNames.Prompt) ?? Options.Prompt,
```

Comment:
Given the usage pattern is always `GetParameter<T>(key) ?? defaultValue`, I think the helper should be `GetParameter<T>(string key, T defaultValue)`

Reply:
For what it’s worth, the GoogleHandler does not use the null-coalescing operator 😝 But yeah, we can do that.

Reply:
Meh, this usage is fine, no need for an additional API.

Reply:
Yeah its not worth adding another public overload of GetParameter, but this would be something that would be done as part of a `ComputeXyz(key, defaultValue)` if we end up doing that